README.md

```diff
@@ -166,7 +166,7 @@ docker run -d -p 2000:2000 \
     --name="promteams" \
     -v /tmp/config.yml:/tmp/config.yml \
     -e CONFIG_FILE="/tmp/config.yml" \
-    docker.io/prometheus-msteams/prometheus-msteams:v1.1.5
+    docker.io/prometheusmsteams/prometheus-msteams:v1.1.5
```

Comment:
I think let's stick with quay.io. The reason I chose it is because of the free security scanner.

https://quay.io/repository/prometheusmsteams/prometheus-msteams?tab=tags